### ISSUE TITLE
Missing information in creating the AcqKnowledge's template graph file

### ISSUE THREAD

**Issue by celprov** [TheAxonLab/hcph-sops#429]
There are '??' that need to be replaced in https://github.com/TheAxonLab/hcph-sops/blob/293976b09c7e001fa8b89f9dd8184169812b3ea9/docs/data-collection/setup.md?plain=1#L122

**Comment by oesteban** [comment]
Good catch - I've thought of opening an issue about this and never got my head around it. Thanks
